pkg/getter/plugingetter.go

```diff
@@ -82,12 +82,12 @@ func (p *pluginGetter) Get(href string) (*bytes.Buffer, error) {
 
 // newPluginGetter constructs a valid plugin getter
 func newPluginGetter(command string, settings cli.EnvSettings, name, base string) Constructor {
```

Comment:
Like my other comment, this may be out of scope, but this seems a little funky. I think if we have a `NewHTTPGetter`, we should also have an exported `NewPluginGetter` instead of this one-off `Constructor` bit that doesn't seem to provide much abstraction value

Reply:
Agreed. Can we do that in a follow-up PR? #5922 would be a good opportunity to do that since it makes changes to the chart downloader's behaviour and how it instantiates the getter.

Reply:
SGTM